src/widgets/main_window.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, settings, **kwargs):
         self.load_history()
         self.return_default_page()
 
+        self.set_default_icon_name('io.github.seadve.Mousai')
```

Comment:
Maybe put this just a space after self.settings = settings

Reply:
I force-pushed it.